cmd/root.go

```diff
@@ -124,8 +118,9 @@ func init() {
 	// will be global for your application.
 	// rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.odo.yaml)")
 
-	rootCmd.PersistentFlags().BoolVarP(&GlobalVerbose, "verbose", "v", false, "Verbose output")
 	rootCmd.PersistentFlags().BoolVar(&GlobalConnectionCheck, "skip-connection-check", false, "Skip cluster check")
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+	pflag.CommandLine.Set("logtostderr", "true")
```

Comment:
Why are we setting this?
Description for `logtostderr` is `log to standard error instead of files (default false)` It is confusing if the message says that default is false and we are changing it here to true.

Reply:
It doesn't work if I remove this line..
Doesn't log anything to console

Reply:
Even openshift seems to be doing this as in https://github.com/openshift/origin/blob/master/tools/depcheck/pkg/cmd/cmd.go#L30